Turf/Turf.swift

```diff
@@ -285,3 +286,61 @@ public struct Polyline {
         return closestCoordinate
     }
 }
+
+public struct Polygon {
+    var coordinates: [CLLocationCoordinate2D]
```

Comment:
Capturing from chat: The coordinates property on a GeoJSON Polygon is represented as a `[[CLLocationCoordate2D]]`